components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1181,6 +1184,35 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
         return dataDO;
     }
 
+    private void getAccessTokenExtendedAttributes(String accessTokenIdentifier, AccessTokenDO dataDO) {
```

Comment:
```suggestion
    private void getAccessTokenExtendedAttributes(String accessTokenIdentifier, AccessTokenDO accessTokenDO) {
```

Reply:
Shall we move this method into a util class?

Reply:
Resolved with ce641ed230aab6181e59cf0439df3e52b17290bb

Reply:
> Shall we move this method into a util class?

It's not getting used by any other class. It's a dependency of Access token retrieval functions.